waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/spark/WriteAsNamedFilesAction.scala

```diff
@@ -51,5 +52,16 @@ case class WriteAsNamedFilesAction(label: String, tempBasePath: Path, destBasePa
 }
 
 object WriteAsNamedFilesAction {
+  def checkTextWriteOptions(numberOfFiles: Int, format: String): Unit = {
+    if (format == "text") {
+      if (numberOfFiles > 1) throw new IllegalArgumentException("When writing text files only 1 file is able to be written")
+    }
+  }
+
+  def fixTextExtension(ext: String): String = ext match {
```

Comment:
Would it make sense to make function name a bit more generic so we can add more case statements for other file types as required?